src/encoded/schemas/ivp_a1.json

```diff
@@ -0,0 +1,396 @@
+        "8 Other primary language (specify)",
+        "9 Unknown"
+      ]
+    },
+    "primlanx": {
+      "title": "12a. Other (specify subject's primary language):",
+      "description":"(Any text or numbers with the exception of single quotes ('), double quotes (\"), ampersands (&), and percentage signs (%).)",
+      "type": "string"
+    },
+    "educ": {
+      "title": "13. Subject's years of education-(Use the codes here to report the level achieved; if an attempted level is not completed, enter the number of years completed.)",
+      "description": "12 = High school or GED,16 = Bachelor's degree,18 = Master's degree,20 = Doctorate,99 = Unknown",
+      "type": "string",
+      "enum": [
+          "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","99"
+        ]
+     
+    },
+    "maristat": {
+      "title": "14. Subject's current marital status:",
+      "type": "string",
+      "enum": [
+        "1 Married",
+        "2 Widowed",
+        "3 Divorced",
+        "4 Separated",
+        "5 Never married (or marriage was annulled)",
+        "6 Living as married/domestic partner",
+        "Other",
+        "9 Unknown"
+      ]
+    },
+    "livsitua": {
+      "title": "15. What is the subject's living situation?",
+      "type": "string",
+      "enum": [
+        "1 Lives alone",
+        "2 Lives with one other person: a spouse or partner",
+        "3 Lives with one other person: a relative, friend, or roommate",
+        "4 Lives with caregiver who is not spouse/partner, relative, or friend",
+        "5 Lives with a group (related or not related) in a private residence",
+        "6 Lives in group home (e.g., assisted living, nursing home, convent)",
+        "9 Unknown"
+      ]
+    },
+    "independ": {
+      "title": "16. What is the subject's level of independence?",
+      "type": "string",
+      "enum": [
+        "1 Able to live independently",
+        "2 Requires some assistance with complex activities",
+        "3 Requires some assistance with basic activities",
+        "4 Completely dependent",
+        "9 Unknown"
+      ]
+    },
+    "residenc": {
+      "title": "17. What is the subject's primary type of residence?",
+      "type": "string",
+      "enum": [
+        "1 Single - or multi-family private residence (apartment, condo, house)",
+        "2 Retirement community or independent group living",
+        "3 Assisted living, adult family home, or boarding home",
+        "4 Skilled nursing facility, nursing home, hospital, or hospice",
+        "Other",
+        "9 Unknown"
+      ]
+    },
+    "zip": {
+      "title": "18. ZIP Code (first three digits) of subject's primary residence:",
+      "description": "(ZIP Code can be blank if unknown, in the range 006-999)",
+      "type": "integer",
+      "minimum": 6,
+      "maximum": 999,
+      "pattern": "[0-9][0-9][0-9]"
```

Comment:
add pattern and make string. 
remove minimm and maximum
"pattern": "[0-9][0-9][6-9]"

Reply:
They have this range 006-999 in code dic. Should we change integer to string to make the pattern work?

Reply:
ya we need to dial in on the pattern but that concept should work

Reply:
![image](https://user-images.githubusercontent.com/38848214/83100647-07764d00-a076-11ea-996f-c7d7292f8b46.png)
Make it work now, but have to be string.